openshift/templates/cakephp-mysql.json

```diff
@@ -153,20 +153,20 @@
         ],
         "replicas": 1,
         "selector": {
-          "name": "cakephp-mysql-example"
+          "name": "${NAME}"
         },
         "template": {
           "metadata": {
-            "name": "cakephp-mysql-example",
+            "name": "${NAME}",
             "labels": {
-              "name": "cakephp-mysql-example"
+              "name": "${NAME}"
             }
           },
           "spec": {
             "containers": [
               {
-                "name": "cakephp-mysql-example",
-                "image": "cakephp-mysql-example",
+                "name": "${NAME}",
+                "image": "${NAME}",
```

Comment:
this can be left as it was, it has no effect since the value gets replaced via the ICT anyway.